src/TypeScript/Analyzer/EslintBridgeAnalyzer.cs

```diff
@@ -135,9 +135,9 @@ private async void ActiveSolutionTracker_ActiveSolutionChanged(object sender, Ac
             await StopServer();
         }
 
-        private async void AnalysisConfigMonitor_ConfigChanged(object sender, EventArgs e)
+        private void AnalysisConfigMonitor_ConfigChanged(object sender, EventArgs e)
         {
-            await StopServer();
+            RequireLinterUpdate();
```

Comment:
In terms of performance, there is no reason to shutdown the entire process when settings are changed, only when the solution is changed.
Restarting the server can take about 10 seconds, so this was very inefficient (and was also causing exceptions if the settings file was saved multiple times, see #2425 )